docs/general/builtins/scalar-functions.rst

```diff
@@ -1055,9 +1055,6 @@ Synopsis::
     operations. In such a case the actual timestamp of each row update is
     returned.
 
-    The return value of expressions ``CURRENT_TIMESTAMP`` and ``CURRENT_TIME``
```

Comment:
We used to use System.currentTimeMillis, see https://github.com/crate/crate/pull/10061/files#diff-f6991c1c4d507accc62ca36dc52adc5029bc955e95c322b2e2c823444f0e2e7bL836-L838 now it's outdated. 
Explicitly mention UTC since we don't support setting any other default timezone, see https://github.com/crate/crate/issues/7196 and https://github.com/crate/crate/issues/10037